.github/workflows/build-arm64-wheels.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - dev
+      - no-simple-arm
```

Comment:
Maybe take this out now that you know it works?

Reply:
Meant to and got distracted.